crates/kas-resvg/src/svg.rs

```diff
@@ -151,6 +151,7 @@ impl_scope! {
 
         fn draw(&mut self, mut draw: DrawMgr) {
             if let Some(id) = self.image.as_ref().map(|h| h.id()) {
+                draw.set_id(self.id());
                 draw.image(self.rect(), id);
             }
         }
```

Comment:
This extra line is what I don't like about the change: `draw` methods must call `set_id` to allow drawing to respond to events (such as hover).

(In this case it's not even necessary since `draw.image(..)` doesn't depend on any event state, but I don't expect users to remember which draw calls *do* depend on event state.)

(And, yes, the "obvious" solution is to pass `self.id` into draw calls, where required. *But* components don't have a `WidgetId` to pass. A few PRs back, `Component::draw` had an extra parameter: `id: &WidgetId`. In order to unify components and widgets, this had to be removed (or added to widgets, and calls to `Widget::draw` updated to use it — which is close to what `Draw::recurse` did).